config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  resources :locations
```

Comment:
Both of these resources lines need to go below the “get ...” on line 5, but before the “root to ...” on 6

Reply:
made the changes